gradle/verification-metadata.xml

```diff
@@ -3,6 +3,9 @@
    <configuration>
       <verify-metadata>false</verify-metadata>
       <verify-signatures>true</verify-signatures>
+      <key-servers>
+         <key-server uri="https://keys.openpgp.org"/>
```

Comment:
Why do you need to open the keyserver now?

Reply:
We got an error on dependency verification that the signing key was not found and we would need to provide the key server where the key is published.